docs/geode-native-docs-cpp/regions/regions.html.md.erb

```diff
@@ -53,20 +53,20 @@ auto cache = CacheFactory()
 
 cache.getPoolManager()
     .createFactory()
-    .addLocator("localhost", 10334)
-    .create("pool");
+    .addLocator("localhost", 40404)
```

Comment:
I would stick with the port 10334 here, which is the default locator port. 40404 is the default server port, so looks very odd to use as a locator port.

Reply:
Corrected locator port (40404 -> 10334) fo both C++ and .NET.